neon-api/src/lib.rs

```diff
@@ -37,6 +37,9 @@ use solana_api::solana_rpc_client_api::response::RpcVersionInfo;
 
 use self::gas_limit_calculator::{GasLimitCalculator, GasLimitError};
 
+/* value from neon-proxy.py */
```

Comment:
I'm not sure we want to "pollute" this repo with mentions of `neon-proxy.py`

Reply:
ok